mfai/torch/models/resnet.py

```diff
@@ -19,7 +19,10 @@ class ResNetEncoder(ResNet):
     - output channels specification of feature tensors (produced by encoder)
     - patching first convolution for arbitrary input channels
     """
-    def __init__(self, out_channels: tuple[int, ...], depth: int = 5, **kwargs: dict[str, Any]):
+
+    def __init__(
+        self, out_channels: tuple[int, ...], depth: int = 5, **kwargs: dict[str, Any]
```

Comment:
mypy is smart enought to know that `**kwargs` means `dict[str, <something>]`. Type hinting `Any` is sufficient.
```suggestion
        self, out_channels: tuple[int, ...], depth: int = 5, **kwargs: Any
```